app/controllers/organizations/staff/user_roles_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Organizations::Staff::UserRolesController < Organizations::BaseController
+  include RoleChangeable
```

Comment:
I was thinking we'd have this as a model concern that is included on the user, so we can call user.change_role. Thoughts? I think this would better sit closer to the user model than in the controller as it operates on the user.

Reply:
Yes that makes more sense.